app/styles/templates/project/index.scss

```diff
@@ -1,14 +1,25 @@
 .project-wrapper {
+  display: flex;
+  flex-direction: column;
   min-height: 50vh;
   width: 100%;
+
+  &--index {
+    flex-direction: row;
+  }
+
+  @include media($sm-screen) {
+    flex-direction: column;
+  }
```

Comment:
I guess one way you could do it is in the `&--index`, you could use 

```
  &--index {
    @include media(>$sm-screen) {
       flex-direction: row;    
    }
  }
```

don't know if the greater than sign can be used like this, but the thought is there.  Then you could remove this line I believe.